libs/ledger-live-common/src/families/casper/bridge/bridgeHelpers/txn.ts

```diff
@@ -22,7 +23,15 @@ export function mapTxToOps(
     const ops: CasperOperation[] = [];
     const { timestamp, caller_public_key, args: txArgs, deploy_hash, error_message } = tx;
     const fromAccount = casperAccountHashFromPublicKey(caller_public_key);
-    const toAccount = txArgs.target.parsed;
+    let toAccount;
+
+    if (txArgs.target.cl_type === "PublicKey") {
+      toAccount = casperAccountHashFromPublicKey(txArgs.target.parsed);
+    } else {
+      toAccount = txArgs.target.parsed;
+    }
+    invariant(toAccount, "toAccount is required");
+    invariant(fromAccount, "fromAccount is required");
```

Comment:
Isn't this a bit harsh? It will cause a crash if any data is missing, I think we would prefer a silent failure, even if it means skipping some operations, the impact would be better for the user. WDYT?

Reply:
I see what you mean, it does feel like a harsh expectation for the addresses to be present. I will fix it in favor of skipping rather than just giving up altogether.

Reply:
I moved the logic of composing an operation inside a try block so if there's any exception raised we can still proceed with other operations. I realized this should have been done sooner 😓